src/test/object.test.ts

```diff
@@ -270,6 +271,54 @@ describe('testing POST object via multipart upload', () => {
       })
     )
   })
+
+  test('should not add row to database if upload fails', async () => {
+    // Mock S3 upload failure.
+    jest.spyOn(S3Backend.prototype, 'uploadObject').mockRejectedValue({
+      name: 'S3ServiceException',
+      message: 'Unknown error',
+      $fault: 'server',
+      $metadata: {
+        httpStatusCode: 500,
+      },
+    })
+
+    process.env.FILE_SIZE_LIMIT = '1'
```

Comment:
Do we require this? Since you are mocking an error response from s3 anyway

Reply:
I'm not sure if it's strictly required, I'm following the other usages of it in the other upload tests in the file.

Reply:
That was added add tests for the case where the file size limits is exceeded. It is not required here.